### PR TITLE
networking for containers and add pumba pilot test

### PR DESCRIPTION
**Pull request by trietnguyen267**

### CODE REVIEW

File: docker-compose.yml [hunk 1]
```diff
@@ -1,5 +1,46 @@
...
+
+  tx-node-1:
+    container_name: tx-node-1
+    image: "node:8"
+    user: "node"
+    working_dir: /home/node/app
+    environment:
+      - NODE_ENV=production
+    volumes:
+      - /Users/triet/centrality/plug-load-test/:/home/node/app
```
Comment (holygits) [inline docker-compose.yml]: This path should be changed so that it can 'run anywhere'

Comment (trietnguyen267) [reply docker-compose.yml]: fixed to mount current directory

File: docker-compose.yml [hunk 2]
```diff
@@ -89,11 +143,19 @@ services:
       - --base-path=/mnt/node
       - --chain=dev
       - --bootnodes=/dns4/boot-node-3/tcp/30333/p2p/QmdoyqhDeYScKjNKZTK9Ru3KugwXxiUBaS2k9CESDbn3Be
-      - --charlie
+      # - --charlie
```
Comment (cowboy-bebug) [inline docker-compose.yml]: we can just remove this line

Comment (trietnguyen267) [reply docker-compose.yml]: done

**Pull request merged**